src/main/webapp/edit-account.html

```diff
@@ -0,0 +1,64 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8">
+    <title>Edit File</title>
+    <link rel="stylesheet" href="assets/edit-account.css">
+    <link rel="stylesheet" href="assets/common.css">
+    <script src="scripts/appscript.js"></script>
+  </head>
+  <body onload="javascript:populateCustomerList()">
+    <div class="drop_menu">
+			<button class="menu_button">Page Menu</button>
+			<div class="dropdown_content">
+				<a href="readfile.html">Read Configuration</a>
```

Comment:
same comment about indentation

Reply:
That is a GH problem. It looks correct on cloudshell.